src/cpu/core_dyn_x86/risc_x86.h

```diff
@@ -39,14 +39,12 @@ static struct {
 
 class GenReg {
 public:
-	GenReg(Bit8u _index) {
-		index=_index;
-		notusable=false;dynreg=0;
-	}
+	Bit8u index;
 	DynReg  * dynreg;
 	Bitu last_used;			//Keeps track of last assigned regs 
-    Bit8u index;
 	bool notusable;
+
+	GenReg(Bit8u _index) : index(_index), dynreg(0), last_used(0), notusable(false) {}
```

Comment:
Another minor comment, if you can install `clang-format` on Windows, we have a `.clang-format` rule file that can auto-format just the changed lines in a commit.

`git clang-format` will format just the change made in the current commit.
Or we have the script, `./script/format-commit.sh` to do this too.

(It will adjust this constructors formatting)

(and some people have figured out how to apply it to a series of commits)
https://stackoverflow.com/questions/43974371/run-git-clang-format-on-series-of-git-commits

But I usually just run it as I go, before checking in my commits.

This is all minor - but nice to get setup (and can be used beyond Staging)

Reply:
> Suggest assigning the `nullptr` to pointer members.
> 
> Here's a nice write up over why `nullptr` gives benefits over `NULL` (or 0): https://embeddedartistry.com/blog/2017/03/08/migrating-from-c-to-c-null-vs-nullptr/

Yeah, I normally use `nullptr` as well. Was just following the original, which used `0`

Reply:
Sorry for the unnecessary info!
The original code usually used NULL, but we still come across some 0's. C++11 and nullptr only came out around 0.74.3 release date - so definitely a nice syntax uplift to the code (throwing pennies in a well.. I know.. but might as well if we're touching it).

Reply:
No worries.

I ended up switching to (mostly) setting default values as suggested in your prior review.